src/main/java/org/jabref/model/entry/Date.java

```diff
@@ -57,15 +58,48 @@ public Date(int year) {
 
     public Date(TemporalAccessor date) {
         this.date = date;
+        endDate = null;
+    }
+
+    /**
+     * Creates a Date from date and endDate.
+     *
+     * @param date the start date
+     * @param endDate the start date
+     * @throws NullPointerException     if DOI/Short DOI is null
```

Comment:
DOI? Seems like you copied the wrong comments

Reply:
My bad, yes, I copied the wrong java comments.
Do you any suggestions to put the JUnit test, or should I create a new .java file for this test?

Thanks so much for the detailed review.

Reply:
1. -
2. In this case, the easiest is to search for the file `DateTest` (located in `src/test/java/org/jabref/model/entry/DateTest.java`). If you ever need to create a completely new test file I'd suggest the [IntelliJ short-cut](https://www.jetbrains.com/help/idea/create-tests.html#create-test), the default option follows the same naming convenient that we are using.
3. Not sure that I understand, but I am guessing there is no question here?

Reply:
There's also some more information on the test conventions we are following in https://devdocs.jabref.org/getting-into-the-code/code-howtos#test-cases

Reply:
thanks very much for the suggestions.

Reply:
I removed the improper comments added the unit test. 
I really appreciate all the constructive and detailed feedback. I felt very lucky to choose JabRef as my first open-source project. Everyone is so nice and helpful.